duckduckgo/iOS
# Block CNAME Cloaking

Task/Issue URL: https://app.asana.com/0/0/1195460247972060/f
Tech Design URL: https://app.asana.com/0/481882893211075/1199166946567557/f
CC: @brindy @catehstn 

**Description**:
This PR uses the CNAMEs section in the block list to block trackers that are using CNAME cloaking via Content Blocking rules.

**Steps to test this PR**:
1. visit duluthtrading.com
1. verify metrics.duluthtrading.com is listed as a blocked tracker
1. visit sky.com
1. verify smetrics.sky.com is listed as a blocked tracker
1. visit other well known websites
1. verify standard functionality is not affected



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
